### PR TITLE
social link styling in profile page

### PR DESCRIPTION
**Pull request by Paarmita**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in a uniquely-named feature branch and has no merge conflicts
* [x] adding a hover effect on social links icon on the profile page
* [x] Minor changes for Social media icons on profile page #2428
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by SidharthBansal** [comment]
Hi @Paarmita Glad to see you back.
Can you please add a GIF so that I can see the css changes.
Thanks for contributing at public labs

File: app/views/users/profile.html.erb [hunk 1]
```diff
@@ -280,6 +280,17 @@
...
+    padding: 50px;
+    transition: transform .2s; 
+    width: 100px;
+    height: 100px;
+    margin: 3 auto;
+}
+
+#link_social_media:hover {
+    transform: scale(1.5); 
+}
```
Comment (seafr) [inline app/views/users/profile.html.erb]: The indentation on lines 289-293 should be moved to the right by 2 spaces to match that of lines 280-288? Do you think you could do that @Paarmita?

Comment (Paarmita) [reply app/views/users/profile.html.erb]: @seafr yeah, i will do it

**Comment by Gauravano** [comment]
hi @Paarmita , just screenshot or GIF of new view required and it's ready to be merged. Thanks.

**Comment by jywarren** [comment]
Hi! It doesn't even have to be a Gif -- just a screenshot of the hovered state would be really helpful. Thanks!!

File: app/views/users/profile.html.erb [hunk 2]
```diff
@@ -280,6 +280,17 @@
   ul.nav-tabs>li a{
     padding: 10px;
   }
+  #link_social_media{
```
Comment (Gauravano) [inline app/views/users/profile.html.erb]: hi @Paarmita, the CSS is implemented on button and changes are expected on profile icons so some changes would be required.

**Comment by Gauravano** [comment]
I have further simplified the issue. See here https://github.com/publiclab/plots2/issues/2428#issuecomment-386720224 . I hope changes are clear now. For any query, please ask.Thanks.

**Comment by jywarren** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by jywarren** [comment]
Thanks, everybody!!!

**Pull request merged**